src/go/package/app/grpc/controllers/package_controller_test.go

```diff
@@ -0,0 +1,38 @@
+package controllers
+
+import (
+	"context"
+	"testing"
+
+	mockstranslation "github.com/goravel/framework/mocks/translation"
+	testingmock "github.com/goravel/framework/testing/mock"
+	"github.com/stretchr/testify/suite"
+
+	mocksservice "market.goravel.dev/package/app/mocks/services"
+)
+
+type PackageControllerSuite struct {
+	suite.Suite
+	ctx                context.Context
+	packageController  *PackageController
+	mockLang           *mockstranslation.Translator
+	mockPackageService *mocksservice.Package
+}
+
+func TestPackageControllerSuite(t *testing.T) {
+	suite.Run(t, new(PackageControllerSuite))
+}
+
+func (s *PackageControllerSuite) SetupTest() {
+	s.ctx = context.Background()
+	mockFactory := testingmock.Factory()
+	s.mockLang = mockFactory.Lang(s.ctx)
+	s.mockPackageService = &mocksservice.Package{}
+	s.packageController = &PackageController{
+		packageService: s.mockPackageService,
+	}
+}
+
+func (s *PackageControllerSuite) TestGetPackage() {
+	// implement me
```

Comment:
Just a reminder.

Reply:
Yeah, added a TODO, will finish it later.